vscode/webviews/components/modelSelectField/ModelSelectField.tsx

```diff
@@ -190,7 +190,7 @@ export const ModelSelectField: React.FunctionComponent<{
                     data-testid="chat-model-popover"
                 >
                     <CommandList
-                        className="model-selector-popover"
+                        className={cn('model-selector-popover', 'tw-max-h-[80vh] tw-overflow-y-auto')}
```

Comment:
```suggestion
                        className="model-selector-popover tw-max-h-[80vh] tw-overflow-y-auto"
```

I think we can just use the string directly here since they are all strings?

Reply:
ty